src/components/Dropdown.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component, PropTypes } from 'react'
+
+class Dropdown extends Component {
+  static propTypes = {
+    children: PropTypes.any,
+    title: PropTypes.string,
+    options: PropTypes.array,
+    isVisible: PropTypes.bool,
+    selected: PropTypes.string,
+    onClick: PropTypes.func,
+  }
+  static defaultProps = {
+    onClick: (item) => { console.log(item) },
```

Comment:
Let's just make this `console.debug(item)` so the user will not see it in the console